source/index.html.md

```diff
@@ -203,6 +204,21 @@ curl "https://write.as/api/posts" \
   -d '{"body": "This is a post.", "title": "My First Post"}'
 ```
 
+```swift
+let client = WFClient(for: "https://write.as")
+let post = WFPost(body: "This is a post.", title: "My First Post")
+client.createPost(post: post) { result in
+  switch result {
+  case .success(let post):
+    print("Post created successfully!")
```

Comment:
My gut says we probably don't need to include sample `print()` statements in our examples, since I think the examples should focus on showing the _library_ at work, and overall be as lean as possible. I do think we could keep the sample comments, though, as they do add something to the examples.

Thoughts?

Reply:
That makes sense to me! Do you think it'd be sufficient to simply leave the comments in place?

Reply:
Yep! Just removing the `print` lines should be perfect.

Reply:
Fixed in fa042d8.